include/internal_constants.h

```diff
@@ -1,6 +1,6 @@
 #pragma once
 
-namespace reinforcement_learning { namespace constant {
+namespace reinforcement_learning { namespace internal_name {
```

Comment:
why do we need this?

Reply:
sorry, let me reformulate:
1) why do we need to rename constants to internal_constants?
2) why is the namespace name internal_name (although file name is internal_constants)?

Reply:
Ask was to use better namespace name than "constant". I am open to better name suggestions.

Reply:
this file looks like not only about constants, but about client config as well. maybe 'config_constants"?

Reply:
updated to suggested